src/common/java/net/minecraftforge/gradle/common/util/runs/GenIDERunsTask.java

```diff
@@ -41,11 +40,9 @@ public void run() {
         MinecraftExtension minecraft = this.getMinecraftExtension().get();
         Project project = this.getProject();
         List<String> additionalClientArgs = this.getAdditionalClientArgs().get();
-        Set<File> minecraftArtifacts = this.getMinecraftArtifacts().getFiles();
-        Set<File> runtimeClasspathArtifacts = this.getRuntimeClasspathArtifacts().getFiles();
 
         runConfigGenerator.createRunConfiguration(minecraft, runConfigurationsDir, project,
-                additionalClientArgs, minecraftArtifacts, runtimeClasspathArtifacts);
+                additionalClientArgs, getMinecraftArtifacts(), getMinecraftArtifacts());
```

Comment:
This doesn't look right. Shouldn't it be `this.getMinecraftArtifacts(), this.getRuntimeClasspathArtifacts()` instead of `this.getMinecraftArtifacts(), this.getMinecraftArtifacts()`?

Reply:
Good spot, thanks!